webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -42,27 +39,20 @@ const Layout = ({
       onExpand={expandLayoutMenus}
     >
       <VerticalNav.Masthead>
-        <VerticalNav.Brand
-          title={data.brand}
-          iconImg={data.logo}
-          href={data.root}
-        />
-        <TaxonomySwitcher
-          currentLocation={currentLocation}
-          locations={data.locations.available_locations || []}
-          onLocationClick={changeLocation}
-          currentOrganization={currentOrganization}
-          organizations={data.orgs.available_organizations || []}
-          onOrgClick={changeOrganization}
-          isLoading={isLoading}
-        />
-        <UserDropdowns
-          notificationUrl={data.notification_url}
-          user={data.user}
-          changeActiveMenu={changeActiveMenu}
-          stopImpersonationUrl={data.stop_impersonation_url}
-          instanceTitle={data.instance_title}
-        />
+        <header className="pf-c-page__header pf-c-page" id="navbar-header">
```

Comment:
Is there a PF4 component with this header? instead of typing it by yourself

Reply:
There is one, it's in the next pr (https://github.com/theforeman/foreman/pull/7809)
 https://github.com/theforeman/foreman/pull/7809/files#diff-68bb61cb73281bdbbb65e8b2cb6aa72995e43b2e6932a7bbe68bb4e871b17cfdR35